app/src/common/shared/org/mozilla/vrbrowser/utils/DeviceType.java

```diff
@@ -8,14 +8,15 @@
 
 public class DeviceType {
     // These values need to match those in Device.h
-    @IntDef(value = { Unknown, OculusGo, OculusQuest, ViveFocusPlus, PicoNeo2, PicoG2 })
+    @IntDef(value = { Unknown, OculusGo, OculusQuest, ViveFocus, ViveFocusPlus, PicoNeo2, PicoG2 })
     public @interface Type {}
     public static final int Unknown = 0;
     public static final int OculusGo = 1;
     public static final int OculusQuest = 2;
-    public static final int ViveFocusPlus = 3;
-    public static final int PicoNeo2 = 4;
-    public static final int PicoG2 = 5;
+    public static final int ViveFocus = 3;
+    public static final int ViveFocusPlus = 4;
+    public static final int PicoNeo2 = 5;
+    public static final int PicoG2 = 7;
```

Comment:
Looks like PicoNeo2 should be 6 and PicoGaze should be 5?

Reply:
They should match what is in `app/src/main/cpp/Device.h`

Reply:
Yep.